src/test/java/com/hillel/items_exchange/controller/AuthControllerIntegrationTest.java

```diff
@@ -168,7 +168,7 @@ void logout_Success_ShouldReturnStatusOk() throws Exception {
                 .contentType(MediaType.APPLICATION_JSON)
                 .accept(MediaType.APPLICATION_JSON))
                 .andDo(print())
-                .andExpect(status().isOk());
+                .andExpect(status().isNoContent());
```

Comment:
Can you rename this test for new expected result?

Reply:
I'll update it. Nice catch!